test/run/ch-run_uidgid.bats

```diff
@@ -100,29 +100,29 @@ setup () {
     # busybox seems to use the same list
     case $status in
         0)      # "success"
-            printf 'RISK\tsuccessful mount\n'
+            info 'RISK    successful mount'
```

Comment:
Here's a case where `info` is not appropriate. The `RISK`/`SAFE` lines are output, not log messages; e.g., it will break if we add the program name to each `info` line, which is a common thing to do for logging.

Also, I want the `\t` in there instead of a literal tab character so it's obvious it's a tab.